plugins/BEdita/API/src/Controller/MediaController.php

```diff
@@ -112,6 +112,12 @@ public function thumbs()
             ->map(function (Stream $stream) use ($options, $preset) {
                 $id = $stream->object_id;
                 $uuid = $stream->uuid;
+                if ($stream->mime_type === 'image/svg+xml') {
```

Comment:
Ok for the first part (moving logic to `Thumbnail::get()`.

This PR wanted to fix a buggy behaviour, with most thumbnail generators... Which generator could be fine with svg?

Reply:
Theoretically using `GlideGenerator` with imagemagick driver should work, but I'm not sure. Anyway you could develop a custom generator to do that.

I think it could be useful in the situation big dimensions SVG when you need to display a little preview.

Maybe at the moment we could skip the generation thinking about that if we needed.